huggle/Localization/en.xml

```diff
@@ -361,12 +364,14 @@
     <string name="main-user-report">Report</string>
     <string name="main-user-ib">Increase badness score</string>
     <string name="main-user-clear-tp">Clear talk page of user</string>
+    <string name="main-user-retrieving-tp">Retrieving TP of %1</string>
```

Comment:
Is TP really a common shortening in this product? It probably shouldn't ."Talk" is just two letters more and translators can decide whether to shorten or not. If needed for consistency with other messages, ok for this patch but please file an issue.